text/0000-Project-Goals-2025h1.md

```diff
@@ -0,0 +1,653 @@
+
+[RFL.com]: https://rust-for-linux.com/
+[RFL#2]: https://github.com/Rust-for-Linux/linux/issues/2
+
+### Project goals
+
+The full slate of project goals are as follows. These goals all have identified owners who will drive the work forward as well as a viable work plan. The goals include asks from the listed Rust teams, which are cataloged in the [reference-level explanation](#reference-level-explanation) section below.
+
+**Invited goals.** Some goals of the goals below are "invited goals", meaning that for that goal to happen we need someone to step up and serve as an owner. To find the invited goals, look for the ![Help wanted][] badge in the table below. Invited goals have reserved capacity for teams and a mentor, so if you are someone looking to help Rust progress, they are a great way to get involved.
+
+| Goal                                                                                                        | Point of contact | Team                                                |
+| ---                                                                                                         | ---              | ---                                                 |
+| ["Stabilizable" prototype for expanded const generics](https://rust-lang.github.io/rust-project-goals/2025h1/min_generic_const_arguments.html)                      | [Boxy][]         | [lang], [types]                                     |
+| [Bring the Async Rust experience closer to parity with sync Rust](https://rust-lang.github.io/rust-project-goals/2025h1/async.html)                                 | [Tyler Mandry][]         | [compiler], [lang], [libs-api], [types]             |
+| [Continue resolving `cargo-semver-checks` blockers for merging into cargo](https://rust-lang.github.io/rust-project-goals/2025h1/cargo-semver-checks.html)          | [Predrag Gruevski][]      | [cargo], [rustdoc]                                  |
+| [Declarative (https://rust-lang.github.io/rust-project-goals/2025h1/`macro_rules!`) macro improvements](macro-improvements.html)                                    | [Josh Triplett][]    | [lang], [wg-macros]                                 |
```

Comment:
```suggestion
| [Declarative (`macro_rules!`) macro improvements](macro-improvements.html)                                    | [Josh Triplett][]    | [lang], [wg-macros]                                 |
```

This had an extra URL added in the title, I updated to match the title of the [linked document](https://rust-lang.github.io/rust-project-goals/2025h1/macro-improvements.html).

Reply:
Thanks. There's something bogus in the script that needs to be fixed.